src/tensor_parallel/factory.py

```diff
@@ -47,7 +47,9 @@ def tensor_parallel(
     :param kwargs: additional keyword arguments passed to TensorParallel init
 
     """
+    model_size = sum(p.numel() for p in module.parameters() if p.requires_grad)
```

Comment:
If it only counts parameters with `p.requires_grad` shouldn't the name mention it somehow? Maybe `num_unfrozen parametes`?

Reply:
done